packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "10.6.2",
+  "version": "11.0.0",
```

Comment:
Do we want to use major changes even when the component is considered experimental or in the "use with caution" phase?

Reply:
As I wrote in a comment above, I waffled on this. No one is using it yet, and you're right it is an experimental component, but I'm not sure it's good practice to change the name of a component and _not_ have it be a major change. I could go either way.

Reply:
I'm in favor of using the correct versioning regardless of usage for consistency. Is there a reason NOT to treat this as a breaking change?